src/test/kotlin/uk/gov/justice/digital/hmpps/hmppstier/integration/AdditionalRequirementsTest.kt

```diff
@@ -11,12 +11,12 @@ class AdditionalRequirementsTest : MockedEndpointsTestBase() {
 
   @Test
   fun `Additional requirements do not cause a processing error`() {
-    val crn = "X333444"
+    val crn = "X833444"
     setupNonCustodialSentence(crn)
     setupRegistrations(registrationsResponse(), crn)
     setupAdditionalRequirements(crn)
     restOfSetupWithMaleOffenderAndSevereNeeds(crn)
     calculateTierFor(crn)
-    expectTierCalculation("A3")
+    expectTierCalculation("A0")
```

Comment:
Surprise!

I wrote this test some time ago and all I wanted was for it to calculate a tier, as this previously threw an error. I should have spent a bit more time looking into what the tier actually was.

I think there's a case for ditching a lot of the existing integration tests and replacing them with BDD. There may be more incorrect behaviour like this which is reinforced by testing.